src/platform/plugins/shared/console/server/routes/api/console/convert_request_to_language/index.ts

```diff
@@ -77,6 +77,12 @@ export const registerConvertRequestRoute = ({
   router.post(
     {
       path: '/api/console/convert_request_to_language',
+      security: {
+        authz: {
+          enabled: false,
+          reason: 'Uses the request-converter library to convert requests to a different language',
```

Comment:
More importantly its not compute intensive....right?

Reply:
I don't know how exactly the transformation happens, if its lightweight (regexes and find-and-replace) or more heavy duty (parsing, AST, etc). Are you concerned about a potential DoS?

Reply:
yes, exactly

Reply:
It might make sense to break this out into its own PR. We should either perf test it OR add some authentication